health-be/note.sql

```diff
@@ -0,0 +1,13 @@
+create table note
```

Comment:
to give an order to all notes, we can add a date or a note number

Reply:
added